src/Settings/Model/GeneralSettings.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Settings\Model;
+
+use MyParcelNL\Pdk\Base\Model\Model;
+
+/**
+ * @property string $apiKey
+ * @property bool   $enableApiLogging
+ * @property bool   $shareCustomerInformation
+ * @property bool   $useSeparateAddressFields
+ * @property bool   $conceptShipments
+ * @property bool   $orderMode
+ * @property string $priceType
+ */
+class GeneralSettings extends Model
+{
+    public const API_KEY                     = 'apiKey';
+    public const CONCEPT_SHIPMENTS           = 'conceptShipments';
+    public const ENABLE_API_LOGGING          = 'enableApiLogging';
+    public const ORDER_MODE                  = 'orderMode';
+    public const PRICE_TYPE                  = 'priceType';
+    public const SHARE_CUSTOMER_INFORMATION  = 'shareCustomerInformation';
+    public const USE_SEPARATE_ADDRESS_FIELDS = 'useSeparateAddressFields';
+
```

Comment:
Ik mis hier: 

- Vraag om startpositie bij printen
- Track & Trace in e-mail
- Track & Trace in Mijn Account
- Toon bezorgdatum in klantcommunicatie
- Verwerk zendingen direct
- Orderstatus automatisering
       - Verander order status na
              - Label printen
              - Exporteren
        - Automatische orderstatus
              - (alle mogelijke orderstatus)
- Bewaar barcode in een notitie
       - Titel voor de barcode
- Automatische export